pkg/server/api/middleware/logger.go

```diff
@@ -7,8 +7,22 @@ import (
 	"github.com/spiffe/spire/pkg/server/api/rpccontext"
 )
 
+const (
+	// serverAPIPrefix is the prefix on the SPIRE server APIs full method names.
+	// We strip it out to keep things concise.
+	serverAPIPrefix = "/spire.api.server."
```

Comment:
Should this constant be here?  it's not used in this file.  Put it in names.go?

Reply:
Whoops, yep.